src/components/MainContainer.js

```diff
@@ -486,14 +481,11 @@ class MainContainer extends Component {
         }));
     };
 
-    handleSummaryStart = (date) => {
-        console.log('start', date)
-        this.setState({summaryStart: date});
-    }
-
-    handleSummaryEnd = (date) => {
-        console.log('end', date)
-        this.setState({summaryEnd: date});
+    handleSummaryDates = (start, end) => {
```

Comment:
if you have super simple methods like this, there's a nice one liner syntax for it.

`handleSummaryDates = (start, end) => this.setState({ summaryStart: start, summaryEnd: end});`

You can do this with all of your simple state setting functions